### PR TITLE
[New GUI] Purge the old gui from Luna Studio

### PR DESCRIPTION
**Pull request by piotrMocz**
### Pull Request Description

**TL;DR:** only the new gui runs when you open LS, `basegl-ui` is an npm package.

- This PR removes the old gui from Luna Studio, cleaning up only a bit more code than absolutely necessary. Also, it now switches to the [npm-packaged `luna-basegl-ui`](https://www.npmjs.com/package/luna-basegl-ui) instead of relying on the submodule structure.
- It addresses https://github.com/luna/luna-studio/issues/914

### Important Notes

- [!] Basegl-ui is now [an npm package](https://www.npmjs.com/package/luna-basegl-ui)
- If you want to still test locally, you need to change the line with `"luna-basegl-ui"` in `luna-studio/luna-studio/atom/package.json` to `"luna-basegl-ui": "file: ../basegl-ui"`
- there will no longer be two gui-s running side-by-side in Luna Studio

### Checklist

- [ ] The documentation has been updated if necessary.
- [x] The code conforms to the [Luna Style Guide](https://github.com/luna/wiki/blob/master/code-style/01.general.md) if it is Haskell.
- [ ] The code has been tested where possible.

### CODE REVIEW

**Comment by piotrMocz** [comment]
I'm very happy to see that these changes made it to the `basegl` branch. I feel obligated to note, however, that with this change you will need to do one of the two things in order to see the `basegl-ui` changes in `luna-studio`:
* commit everything inside the `basegl-ui` repo and run `npm publish`
* change the `package.json` (`luna-studio/atom/package.json`) to take the `luna-basegl-ui` package locally

It's not a big problem, but everyone needs to be aware of that ;)

**Comment by wdanilo** [comment]
@piotrMocz As I understand your comment - it's because `luna-basegl-ui` is just an npm package and thus you need ot either fork it locally or just publish the changes to npm to update all dependent packages?

**Comment by piotrMocz** [comment]
@wdanilo yes, that's precisely right ;)

**Comment by wdanilo** [comment]
@piotrMocz awesome, it's the right design. Good job guys! <3
